njoy_processor/Utils_ReadNJOYOutput.py

```diff
@@ -360,38 +368,68 @@ def ReadNJOYfile(njoy_filename = "output", verbose = False):
                 cross_sections["delayed_chi"] = \
                     ProcessDelayedChi(nL, file_lines)
 
+###---------matrix ------------------------------------------------------------
+            # caveat: the pp values from transfer(mf26) = 2x the pp from the xsec(mf23)
+            # caveat: the n,2n values from transfer(mf8/mt16) = 2x the n,2n from the xsec(mf3/mt16)
             if words[num_words - 1] == "matrix":
-                transfer_matrices[words[num_words - 3]] = \
+                particule_type = words[num_words - 2]
+                reaction_type  = words[num_words - 3]
+                if particule_type=='free-gas':
+                    particule_type = 'neutron'
+                    reaction_type  = 'free-gas'
+                transfer_matrices[particule_type][reaction_type] = \
                     ProcessTransferMatrix(nL, file_lines)
```

Comment:
This bugs with the S(alpha,beta) examples.

Reply:
The original code produces in the console

![image](https://user-images.githubusercontent.com/573476/122655530-91c59f00-d118-11eb-9c53-3140d0ad9d21.png)

However, the code has those lines

![image](https://user-images.githubusercontent.com/573476/122655553-c3d70100-d118-11eb-88e4-962465b32503.png)

The example does not use that s_ab keyword in the cross section dict, only in the transfer dict, so I will leave it as is as it causes for issues (for lacking of testing)